python/3d/auto_generated/qgs3dmapsettings.sip.in

```diff
@@ -79,6 +83,29 @@ to improve the precision.
     QgsVector3D origin() const;
 %Docstring
 Returns coordinates in map CRS at which 3D scene has origin (0,0,0)
+%End
+
+    void setOrientation( double newOrientationZ );
```

Comment:
```suggestion
    void setOrientation( double orientation );
```

Although I'd lean toward setZRotation instead -- we use rotation for this purpose rather than orientation in the API. Orientation is for ring orientation (clockwise / counter clockwise), or page orientation (landscape / portrait)

Reply:
`rotation` is now used everywhere.